CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
   - Alpine: 3.14.0 -> 3.14 (#28)
   - Golang: 1.16.5 -> 1.16 (#28)
 
+- Changed docker-build scripts for easier windows building. (#39)
```

Comment:
nit: Could use the filename "Dockerfile" instead of "docker-build scripts" to be more precise. IMO I understand "Dockerfile" better

```suggestion
- Changed Dockerfile for easier windows building. (#39)
```